src/lib/pages/home/components/TopDecorations.tsx

```diff
@@ -0,0 +1,32 @@
+import { Image } from "@chakra-ui/react";
+
+export const TopDecorations = ({ chain }: { chain: string }) => {
```

Comment:
Avoid hardcoding the chain's information here. Please move this to the default theme's config instead.